feature-libs/customer-ticketing/components/details/customer-ticketing-details/customer-ticketing-details.component.html

```diff
@@ -0,0 +1,51 @@
+<!-- TODO:CXEC-17553 (Brian / Parth) add spinner until the data is loaded  -->
+<ng-container *ngIf="ticketDetails$ | async as ticket; else spinner">
+  <div class="cx-ticket-details">
+    <div class="container">
+      <div class="cx-details-card">
+        <cx-card
+          [content]="
+            prepareCardContent(ticket.id!, 'customerTicketing.ticketId') | async
+          "
+        ></cx-card>
+      </div>
+
+      <div class="cx-details-card">
+        <cx-card
+          [content]="
+            prepareCardContent(
+              (ticket.createdAt | cxDate: dateFormat) ?? dateFormat,
```

Comment:
@Parthlakhani 

let's discuss these changes to solve strict typing of templates for line 17, 28 and 39. 

You should always avoid `! to assert that it will never be null/undefined`. Let's see the errors and discuss other ways to circumvent it.